draft-patton-cfrg-vdaf.md

```diff
@@ -861,15 +865,21 @@ pseudorandom field elements.
 ~~~
 # Derive a new seed.
```

Comment:
Can you add some information for why this is needed? Also please add that the seed MUST have high entropy.

Reply:
The intended usage is explained above:
> Each `Prg` has two derived class methods. The first is used to derive a fresh seed from an existing one.

If you want to see this get used, search for `Prg.derive` in the prio3 section.

High entropy ins't going to be sufficient for all PRGs, in particular for those constructed from a pseudorandom permutation (e.g., AES). I stipulated the same requirement for securely generating seeds as above.

Reply:
The current documentation sounds good, thanks!